activemodel/lib/active_model.rb

```diff
@@ -2,4 +2,4 @@
 # disabled until they're tested
 # require 'active_model/callbacks'
 # require 'active_model/validations'
-require 'active_model/base'
\ No newline at end of file
+require 'active_model/base'
```

Comment:
Whoops! This newline can technically go away or we can shim in the require of rails2_ruby2/active_model extensions.